wfl/Controllers/MenuSelectionsController.cs

```diff
@@ -0,0 +1,56 @@
+﻿using System;
+using System.Web.Http;
+using System.Threading.Tasks;
+using System.Linq;
+using System.Collections.Generic;
+using Symphono.Wfl.Models;
+using Symphono.Wfl.Database;
+
+namespace Symphono.Wfl.Controllers
+{
+    [RoutePrefix("api/menu-selection")]
+    public class MenuSelectionsController : ApiController
+    {
+        IDBManager DBManager;
```

Comment:
Yes, I can see certain methods and similarities that are common to the controllers.